app/controllers/categories_controller.rb

```diff
@@ -1,29 +1,14 @@
 class CategoriesController < ApplicationController
 
   def index
-    @categories  = Category.all
-    @objects     = RestoredObject.all
-    half_size    = @categories.count/2+1
-    @first_half  = @categories[0..half_size]
-    @second_half = @categories[half_size..(@categories.count-1)]
-    if @objects.count >= 6
-      @first_sixs_elements      = @objects[0..5]
-      if @objects.count >= 12
-        @second_three_elements  = @objects[5..7]
-      else
-        @second_three_elements  = @objects[5..(@categories.count-1)]
-      end
-    else
-      @first_sixs_elements      = @objects[0..(@categories.count-1)]
-      @second_three_elements    = nil
-    end
 
-    @random_outstanding_object = getRandomObject
   end
 
   def show
-    @category  = Category.find(params[:id])
-    @objects   = @category.restored_objects.page params[:page]
+    @categories  = Category.all
+    @objects     = RestoredObject.all
+    @category    = Category.find(params[:id])
+    @category_id = params[:id]
```

Comment:
We don't need all that info. We're using only `@category` and `@objects` as in the master branch code

Reply:
Sure, I didn't realize that variables were not used. 🙃🙃🙃